static/manifest.json

```diff
@@ -4,9 +4,8 @@
   "description": "Learn AMP by Examples",
   "start_url": "https://ampbyexample.com",
   "display": "standalone",
```

Comment:
One thought:  Since you're using standalone display, Chrome is unable to display a back button.  Some devices don't include back buttons, and ABE doesn't have one, which means that on some mobile devices, it's possible that a user can hit a part of the stack that's not reversible.

As it stands, I'm not sure if the current application flow has any bit that's not accessible by the menu, or that doesn't have access to the menu.  I'm not sure what happens when navigating to GitHub when in standalone mode.  I can't easily test that from staging.